test-case-reporting/test/test_result_record.test.ts

```diff
@@ -107,7 +107,7 @@ describe('TestResultRecord', () => {
       });
     });
 
-    it("throws an error if the build doesn't exist");
+    it.todo("throws an error if the build doesn't exist");
```

Comment:
:tada:

Reply:
:D

Reply:
Can this test be completed in this PR, or is it blocked on something?

Reply:
Unresolving conversation to respond:

I don't believe there's anything blocking this, but I don't want to make this PR any larger than it currently is -- I'll make an issue & add a TODO with the issue number; does that sound good?

Reply:
Done adding the TODO.